assets/home/main.js

```diff
@@ -17,9 +17,20 @@ import GetHelp from './sections/get-help';
 import SenseiGuides from './sections/sensei-guides';
 import LatestNews from './sections/latest-news';
 import Extensions from './sections/extensions';
+import { useEffect, useState } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
 
 const Main = () => {
 	useSenseiColorTheme();
+	const [ data, setData ] = useState( {} );
+
+	useEffect( async () => {
+		const remoteData = await apiFetch( {
+			path: '/sensei-internal/v1/home',
```

Comment:
I think we can preload this JSON in order to optimize this page. :)

To do that, you could import `/assets/shared/data/api-fetch-preloaded-once` in the JS, and call the `Sensei()->assets->preload_data` after the enqueue in the PHP.

Reply:
Done in 0fdaada6c. Thanks for the tip!